packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -150,16 +151,15 @@ export async function beaconHandlerInit(args: IBeaconArgs & IGlobalArgs) {
 }
 
 export function overwriteEnrWithCliArgs(enr: ENR, args: IBeaconArgs): void {
-  // TODO: Not sure if we should propagate this options to the ENR
-  if (args.port != null) enr.tcp = args.port;
-  // TODO: reenable this once we fix the below discv5 issue
+  // TODO: Not sure if we should propagate port/defaultP2pPort options to the ENR
+  enr.tcp = args["enr.tcp"] ?? args.port ?? defaultP2pPort;
+  // TODO: add `port`/`defaultP2pPort` port as backup as well once we
+  // fix the below discv5 issue
+  //
   // See https://github.com/ChainSafe/discv5/issues/201
-  // if (args.port != null) enr.udp = args.port;
-  if (args.discoveryPort != null) enr.udp = args.discoveryPort;
-
+  const udpPort = args["enr.udp"] ?? args.discoveryPort;
+  if (udpPort != null) enr.udp = udpPort;
```

Comment:
we should not set udp to enr until the above discv5 issue is resolved

Reply:
this is just previous logic rehashed, 
```
 if (args.discoveryPort != null) enr.udp = args.discoveryPort;
if (args["enr.udp"] != null) enr.udp = args["enr.udp"];
```
so it will not assign udp if  `discoveryPort` and `enr.udp` args are not externally provided

Reply:
thanks for the clarification @g11tech  👍